src/frontend/Sider/Behandling/Fanemeny/HamburgermenyBehandling.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { useFlag } from '@unleash/proxy-client-react';
+
+import { useBehandling } from '../../../context/BehandlingContext';
+import { Hamburgermeny, MenyItem } from '../../../komponenter/Hamburgermeny/Hamburgermeny';
+import { BehandlingType } from '../../../typer/behandling/behandlingType';
+import { Steg } from '../../../typer/behandling/steg';
+import { Toggle } from '../../../utils/toggles';
+
+export const HamburgermenyBehandling = () => {
+    const { behandling, settVisRedigerGrunnlagFomAdmin, behandlingErRedigerbar } = useBehandling();
+    const kanRedigereGrunnlagFom = useFlag(Toggle.KAN_REDIGERE_GRUNNLAG_FOM);
+
+    const items: MenyItem[] = [
+        ...(behandling.type === BehandlingType.FØRSTEGANGSBEHANDLING &&
+        behandling.steg === Steg.INNGANGSVILKÅR &&
+        kanRedigereGrunnlagFom
+            ? [
+                  {
+                      tekst: 'Rediger dato saksopplysninger hentes fra',
+                      onClick: () => settVisRedigerGrunnlagFomAdmin(true),
+                  },
+              ]
+            : []),
+    ];
```

Comment:
Her kan vi assigne items direkte uten å bruke spread operatoren. Det gjør det litt enklere å lese: 

```
const items: MenyItem[] = 
        behandling.type === BehandlingType.FØRSTEGANGSBEHANDLING &&
        behandling.steg === Steg.INNGANGSVILKÅR &&
        kanRedigereGrunnlagFom
            ? [
                  {
                      tekst: 'Rediger dato saksopplysninger hentes fra',
                      onClick: () => settVisRedigerGrunnlagFomAdmin(true),
                  },
              ]
            : [])
  
   ```

Reply:
Det er behov for å legge till flere items i listen på sikt

Reply:
Endret til en const med push til listen
https://github.com/navikt/tilleggsstonader-sak-frontend/pull/664/commits/49bc3adabeebfa35310c39a7f0a3e7ffddc73985